utils/EA/config.py

```diff
@@ -1,7 +1,7 @@
 ### General ###
 maximization = True
 runs = 1
-generations = 50
+generations = 5
 population_size = 100
 chromo_size = 100
 prob_mutation = 1/generations
```

Comment:
shouldn't we do 1/chromo_size

with generations  = 5 I suppose this to be very random